### PR TITLE
Level des

### PR DESCRIPTION
**Pull request by duriumm**
Level design started by reworking auto tiling and some bug fixes

- Item tooltip wont persist when closing/opening inventory since we do cancel tooltip on every closing of inventory
- Removed unused files along with previous player area
- Autotiles for land with water edges, dirt with grass edges and Random tile for flowerbed
- Equipped items now carry over between scenes

- (Bug) In first scene "SampleScene" there is a bug that disables tooltip hovering on helmet equipment. Only when equipped

### CODE REVIEW

File: Pixel Game/Assets/Scripts/PlayerScripts/PlayerInventory.cs [hunk 1]
```diff
@@ -63,8 +63,9 @@ void Start()
     {
         mainCamera = GameObject.FindWithTag("MainCamera");
         LoadInvGameObjectOnStartScene();
+        LoadEquipmentOnStartScene();
```
Comment (yousernaym) [inline Pixel Game/Assets/Scripts/PlayerScripts/PlayerInventory.cs:66]: A little bit of nitpicking here, but I think `OnStartScene` in the names is overkill. Maybe the methods will be needed elsewhere in the future, then the names would be misleading.

File: Pixel Game/Assets/Scripts/SceneTransition.cs [hunk 2]
```diff
@@ -64,8 +66,9 @@ public void OnTriggerEnter2D(Collider2D collision)
 
             // Here we save all data to our database (dataToPassBetweenScenesGameObject) on entering a new scene
             dataToPassBetweenScenes.playerHealthDB = myPlayerObject.GetComponent<PlayerHealth>().Hp;
-            invManager.GetComponent<PlayerInventory>().SaveInvGameObjectsOnSceneChange();
-            dataToPassBetweenScenes.playerMoneyDB = invManager.GetComponent<PlayerInventory>().playerInvMoney;
+            playerInventory.SaveInvGameObjectsOnSceneChange();
+            playerInventory.SaveEquipmentOnSceneChange(); // TODO: Only saves to inventory, make it equip said items aswell
```
Comment (yousernaym) [inline Pixel Game/Assets/Scripts/SceneTransition.cs:70]: Same name concern here

File: Pixel Game/Assets/Scripts/PlayerScripts/PlayerInventory.cs (hunk 1 above)
Comment (duriumm) [reply Pixel Game/Assets/Scripts/PlayerScripts/PlayerInventory.cs:66]: Hmm.. I wanted it to be as clear as possible to not get the function mixed up somehow.. 
Not sure if i agree to 100% with you here!

Comment (yousernaym) [reply Pixel Game/Assets/Scripts/PlayerScripts/PlayerInventory.cs:66]: Ok, if you feel strongly about it that's fine, let's leave it like this and only change it if we must.

**Pull request merged**